ippon/test_registration_views.py

```diff
@@ -31,12 +40,22 @@ def test_register_unique_user_with_valid_payload_creates_new_user(self, email_us
             message="You have been successfully registered in ippon with username {}".format(
                 self.valid_payload["username"]))
 
-    def test_register_with_not_unique_username_does_not_create_new_user(self):
+    def test_non_unique_username_response(self):
         User.objects.create_user(
             username=self.valid_payload["username"],
             email='a' + self.valid_payload["email"],
             password=self.valid_payload["password"])
         response = self.client.post(reverse('register-user'),
                                     data=json.dumps(self.valid_payload),
                                     content_type='application/json')
+
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertEqual(response.content, self.unique_name_error)
+
+    def test_invalid_email_format_response(self):
```

Comment:
The same as above. Currently we know probable action of this test, but no clue about assertion.